core/roboconf-target-iaas-vmware/src/main/java/net/roboconf/target/vmware/internal/VmWareMachineConfigurator.java

```diff
@@ -113,7 +114,11 @@ public boolean configure() throws TargetException {
 			return true;
 
 		} catch( Exception e ) {
-			throw new TargetException( e );
+			if(++ this.vmwareToolsCheckCount < 20) {
+				this.logger.fine("VMWare tools not yet started... check #" + this.vmwareToolsCheckCount + " failed");
+				return false;
+			}
+			else throw new TargetException( e );
```

Comment:
I did not mean throwing an exception was useless.
I meant putting the **else** keyword before throwing an exception is useless.